### DISCUSSION TITLE
Configure Link component globally

### DISCUSSION THREAD

**Discussion by jgb-solutions** [remix-run/remix#3887] [Q&A] [answered]
I tend to use `prefetch="intent"` on all the links of a project. I'd like to set it once. Is there a way to do that globally with a config or should I create a wrapper Link component for that?

**Comment by kiliman** [comment, marked as answer]
There’s no global option so I would create a wrapper function.

**Reply by jgb-solutions** [reply]
Thanks @kiliman!
